src/apps/reservation-list/reservation-material/reservation-info.tsx

```diff
@@ -53,58 +55,83 @@ const ReservationInfo: FC<ReservationInfoProps> = ({
       >
-        <img src={check} alt="" />
-        {t("readyText")}
+        <div className="counter__value" aria-hidden="true">
+          <img src={check} alt="" />
+          <span className="counter__label">
+            {t("reservationListReadyText")}
+          </span>
+        </div>
       </ReservationStatus>
     );
   }
 
   if (state === "reserved" && pickupBranch && numberInQueue && expiryDate) {
-    // todo string interpolation
     const numberInLineLabel =
       numberInQueue === 1
-        ? t("youAreFirstInQueueText")
-        : `${t("youAreNumberInQueueText")} ${numberInQueue - 1}`;
+        ? t("reservationListFirstInQueueText")
+        : t("reservationListNumberInQueueText", {
+            count: numberInQueue - 1,
+            placeholders: { "@count": numberInQueue - 1 }
+          });
 
     return (
       <ReservationStatus
-        color={colors.default as string}
+        ariaLabel={t("reservationListStatusIconQueuedAriaLabelText", {
+          count: numberInQueue,
+          placeholders: { "@count": numberInQueue }
+        })}
         // The decision regarding this is, that if the user is number 4
         // in the queue for a material, the "percent-wheel-thing" should be 1/4 full.
         percent={(1 / numberInQueue) * 100}
-        expiresSoonLabel={
-          daysBetweenTodayAndDate(expiryDate) <= thresholds.warning
-            ? t("expiresSoonText")
-            : ""
-        }
         label={numberInLineLabel}
       >
-        <span className="counter__value">{numberInQueue}</span>
-        <span className="counter__label">{t("inLineText")}</span>
+        {/* I am not using string interpolation here because of styling */}
+        {/* if somehow it is possible to break text in one div into two lines */}
+        {/* where the first line has another font size AND is only the first "word" */}
+        {/* then this should be changed to do that */}
+        <span className="counter__value" aria-hidden="true">
+          {numberInQueue}
+        </span>
+        <span className="counter__label" aria-hidden="true">
+          {t("reservationListInQueueText")}
+        </span>
       </ReservationStatus>
     );
   }
 
   if (state === "reserved" && !pickupBranch && pickupDeadline) {
+    const daysBetweenTodayAndPickup = daysBetweenTodayAndDate(pickupDeadline);
     return (
-      // todo string interpolation
       <ReservationStatus
-        color={colors.default as string}
+        ariaLabel={t("reservationListStatusIconReadyInAriaLabelText", {
+          count: daysBetweenTodayAndPickup,
+          placeholders: { "@count": daysBetweenTodayAndPickup }
+        })}
         percent={daysBetweenTodayAndDate(pickupDeadline) / 100}
-        label={`${t("canBeLoanedInText")} ${daysBetweenTodayAndDate(
-          pickupDeadline
-        )}`}
+        label={t("reservationListAvailableInText", {
+          placeholders: { "@count": daysBetweenTodayAndDate(pickupDeadline) }
```

Comment:
Just if you do not know it. You can use whatever placeholder when it is a number. So in this case it could be: `@days`

Reply:
yes.